### DISCUSSION TITLE
2024-11-20 Meetup @ SPACELEND

### DISCUSSION THREAD

**Discussion by schadesa** [cloud-native-austria/cncf-graz#8] [General]
- [x] date: 2024-11-21
- [x] host: @DrackThor 
- [x] speaker: Christian Haintz
- [ ] talk: Just one command to setup Karpenter Cluster Auto Scaler with Pulumi
- [ ] sponsor drinks: 
- [ ] sponsor food/snacks: 
- [ ] venue: SPACELEND
- [ ] bevy event: @schadesa  
- [ ] meetup event: @schadesa 
- [ ] linkedin posts: @schadesa 
- [ ] cloud native newsflash - @kevin-schmid / @DrackThor 
- [ ] useful tools - @sadoMasupilami 
- [ ] pubquiz - @schadesa 
- [ ] SWAG/prices - @skillpoint-dev 

Meetup Agenda:

- Welcome from the Hosts
- RSVP awareness -> Jetbrains Licenses!
- KCD Recap @DrackThor?
- CNCF Newsflash?
- Useful Tools?
- Talk
- Quiz

talk abstract:
```
Christian Haintz from Carrot & Company is presenting how easy it can be to set up an AWS EKS Cluster with Karpenter as a Cluster Autoscaler when you have coded the infrastructure setup in Pulumi. He talks about how he approached the transition from CloudFormation to the Pulumi Python IaC Stack and how the Karpenter setup works.
```

**Comment by DrackThor** [comment]
done
